src/Playroom/Snippets/Snippets.tsx

```diff
@@ -94,15 +52,27 @@ export default ({ snippets, onHighlight, onClose }: Props) => {
     },
     50
   );
-  const debounceScrollToHighlighted = useDebouncedCallback(
-    scrollToHighlightedSnippet,
-    50
-  );
+
   const filteredSnippets = useMemo(
     () => filterSnippetsForTerm(snippets, searchTerm),
     [searchTerm, snippets]
   );
 
+  const highlightedItemId =
+    typeof highlightedIndex === 'number'
+      ? getSnippetId(
+          filteredSnippets[highlightedIndex].group,
+          filteredSnippets[highlightedIndex].name,
+          highlightedIndex
+        )
+      : null;
+
+  const highlightedItem = highlightedItemId
+    ? document.getElementById(highlightedItemId)
+    : null;
```

Comment:
Maybe this could be more elegant, but I'm not sure the best way

Reply:
Yeah agree we could refine this a bit. My suggestion would be for the `getSnippetId` function to take a `Snippet` so something like:
```ts
function getIdForSnippet(snippet: Snippet, index: number) {
  return `${snippet.group}_${snippet.name}_${index}`;
}
```
then the above could be more like:
```ts
if (typeof highlightedIndex === 'number') {
  const highlightedItem = document.getElementById(
    getIdForSnippet(filteredSnippets[highlightedIndex], index)
  );

  highlightedItem?.scrollIntoView({ block: 'nearest' });
}
```